src/docs/platforms/javascript/index.mdx

```diff
@@ -122,23 +122,27 @@ Upload source maps using either our [Webpack plugin](./config/sourcemaps/#webpac
 
 Performance Monitoring helps you see everything from macro-level metrics to micro-level spans, and you’ll be able to cross-reference transactions with related issues, customize queries based on your personal needs, and substantially more.
 
-To get started with performance monitoring using Sentry’s JavaScript SDK, first install the `@sentry/browser` and `@sentry/tracing` packages:
+To get started with performance monitoring using Sentry’s JavaScript SDK, install both the `@sentry/browser` and `@sentry/tracing` packages, or link directly to our combined minified bundle via CDN:
 
-```bash
+```bash {tabTitle: ESM}
 # Using yarn
 $ yarn add @sentry/browser @sentry/tracing
 
 # Using npm
 $ npm install @sentry/browser @sentry/tracing
 ```
 
-Alternatively, instead of `npm` packages, you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/tracing`:
-
-<JsCdnTag apm={true} />
```

Comment:
Wasn't sure how to get the tabbed experience while using this `<JsCdnTag/>` component. Open to ideas if someone can direct me.

Reply:
for anyone else: you cant, what ben did here is the right solution